aws-s3-transfer-manager/tests/test_utils.rs

```diff
@@ -0,0 +1,31 @@
+use aws_s3_transfer_manager::{error::Error, operation::download::DownloadHandle};
+use bytes::{BufMut, Bytes, BytesMut};
+use tracing_subscriber::EnvFilter;
+
+/// drain/consume the body
+pub async fn drain(handle: &mut DownloadHandle) -> Result<Bytes, Error> {
```

Comment:
Yuki added a [test-common](https://github.com/awslabs/aws-s3-transfer-manager-rs/tree/main/aws-s3-transfer-manager/test-common) package that this could live in